### PR TITLE
Responsive Changes

### PR DESCRIPTION
**Pull request by nl32**
This pr adds tweaks to the layout and styling at different breakpoints to make it more usable on smaller screens and phones. 
this makes most of the core features more usable on mobile, however a lot of it doesn't look perfect 🫣, so comments are greatly appreciated.
One large issue is that the directory grid height shifts whenever new orgs load in.

### CODE REVIEW

**Review by RubenOlano — CHANGES_REQUESTED** [review]
Looks good but some text looks a little to large on smaller screens

File: src/app/auth/page.tsx [hunk 1]
```diff
@@ -23,12 +23,14 @@ export default async function Auth({
...
         </div>
         <div className="flex flex-col gap-16">
-          <div className="z-10 flex w-fit">
+          <div className="absolute inset-5 z-10 flex w-fit sm:static">
             <BackButton />
           </div>
           <div className="z-10 flex flex-col items-center justify-center space-y-12">
-            <h1 className="text-6xl font-bold text-white">Sign In / Sign Up</h1>
-            <div className="flex w-full flex-row justify-center gap-x-4 py-2.5">
+            <h1 className="text-center text-6xl font-bold text-white sm:text-left">
```
[the remark this answers is not in the document]
Comment (nl32) [reply src/app/auth/page.tsx:30]: I made the text bigger on the buttons, since it helps readability and fills in the gaps. let me know what you think.

**Comment by nl32** [comment]
> Looks good but some text looks a little to large on smaller screens
I made a lot of text everywhere bigger, let me know if you see any other pain points.

File: src/app/auth/page.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (RubenOlano) [reply src/app/auth/page.tsx:30]: Looks good, only thing that sticks out to me is that the button text is not centered but hopefully shouldn't be a crazy fix

**Comment by Jake3231** [comment]
@RubenOlano @nl32 What is the expected usability for this PR? "Usable" vs "Polished"? Overall looks great, but there's a fee places that need work but are certainly usable right now.

**Comment by nl32** [comment]
> What is the expected usability for this PR? "Usable" vs "Polished"? Overall looks great, but there's a fee places that need work but are certainly usable right now.

The goal was largely to get rid overflow issues on smaller screens, improve readability on smaller screen sizes, and that core features like the directory, events, settings, and all their functionality can be used on mobile, Which I think this pr addresses.
The main areas that I see as having issues are the carrousel, club page header, and events page, which each are being addressed in their own tickets.

@Jake3231 if you see anything else you think needs adjustment or polishing I'd love to hear it.

**Comment by RubenOlano** [comment]
Only thing I got is the login button stuff not being centered, other than that I'm happy

**Comment by Jake3231** [comment]
> > What is the expected usability for this PR? "Usable" vs "Polished"? Overall looks great, but there's a fee places that need work but are certainly usable right now.
> 
> The goal was largely to get rid overflow issues on smaller screens, improve readability on smaller screen sizes, and that core features like the directory, events, settings, and all their functionality can be used on mobile, Which I think this pr addresses. The main areas that I see as having issues are the carrousel, club page header, and events page, which each are being addressed in their own tickets.
> 
> @Jake3231 if you see anything else you think needs adjustment or polishing I'd love to hear it.

The biggest thing is the club page on mobile. If that's a known issue that's being addressed elsewhere, it looks good to me!

**Review by RubenOlano — APPROVED** [review]

**Pull request merged**